dist/alpine/Dockerfile

```
@@ -11,6 +11,7 @@ COPY --chown=anaconda:anaconda ./dist/alpine/apk_requirements.txt \
 COPY --chown=anaconda:anaconda ./utilities/djlab.py ./utilities/setup.py /main/
 RUN \
 	umask u+rwx,g+rwx,o-rwx && \
+	chmod 664 /tmp/djlab_config.yaml && \
```

This is more for styling but a bit easier to read/review with this syntax:

```suggestion
	chmod u=rw,g=rw,o=r /tmp/djlab_config.yaml && \
```